src/celscript/lua/celx_celestia.cpp

```diff
@@ -2246,6 +2246,7 @@ static int celestia_verbosity(lua_State* l)
 }
 
 
+#ifdef USE_MINIAUDIO
```

Comment:
this should be inside the function body

Reply:
The warning here is "unused function". This function is only called when `USE_MINIAUDIO` is enabled.

Reply:
yes, correct. due to its name i decided that it's part of celx.